### DISCUSSION TITLE
710

### DISCUSSION THREAD

**Discussion by giscus** [jojoldu/blog-comments#891] [Comments]
# 710

최근에 배민에서 세미나 발표를 했다. 끝나고 별도의 자리에서 와주신 분들과 한분씩 고민 상담을 진행했다. (사실 책 사인하는 시간이였는데… 사인하면서 하나 궁금한점이 있다고 하시면서 본격 고민 상담 시간..) 그때 가장 많이 나온 질문이 "나는 주니어인데 좋은 개발팀에서 성장하고 싶다. 지금 팀의 개발 문화를 어떻게 변화시킬 수 있는지, 어떻게 하면 팀원들을 설득시킬 수 있냐"는 것이다. 고민을 이야기하신 분들 대부분이 다음과 같은 팀에 속해있었다. 코드 리뷰를 원하지 않는 팀원 테스트 코드가 불필요하다는 팀원 퇴근 후 공부를 원하지 않는 팀원 본인은 팀원들과 스터디도 같이 하고 싶고, 코드리뷰도 하고 싶고, 테스트 코드도 작성해보면서 제품을 개발해보고 싶다는 것이다. 물론 위와 같은 팀원들이 나쁘다는 ..

https://jojoldu.tistory.com/710

**Comment by smw0807** [comment]
좋은 내용이네요.

**Comment by juhwano** [comment]
-개발자는 누구나 공부하는 줄 알았습니다.
-개발자는 누구나 개발을 좋아하는 줄 알았습니다.
-개발자는 누구나 자신의 지식을 다른 사람과 공유하는 걸 좋아하는 줄 알았습니다.
-개발자는 누구나 논리적인 줄 알았습니다.

**Comment by basejb** [comment]
사내에서 만족을 못하다보니 외부적으로 사이드로 채워나가고 있었는데 정말 느껴지는게 많습니다 ㅎㅎ
유튜브에 나오시는거 찾아보곤 하는데 어제 카페에서 들리는 목소리가 너무 익숙해서 돌아봤는데 이동욱 CTO님이라 정말 깜짝 놀랐습니다.
갑작스러운 사진촬영 응해주셔서 죄송하고 감사합니다!

**Comment by mystyle2006** [comment]
공감이 많이 되는 내용입니다.
